.docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:latest
+MAINTAINER appbase.io <info@appbase.io>
+
+ENV APP_USER  "app"
+ENV APP_DIR "/project"
+
+RUN useradd --user-group --create-home --shell /bin/false $APP_USER
+
+WORKDIR $APP_DIR
+
+RUN npm install -g bower
+RUN npm install -g typescript
```

Comment:
Installing typescript is not necessary for dejavu. However, it does require installing gulp.